web-ui/src/components/volunteer/VolunteerRelationships.jsx

```diff
@@ -0,0 +1,468 @@
+import { format } from 'date-fns';
+import PropTypes from 'prop-types';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+
+import { AddCircleOutline, Delete, Edit, Handshake } from '@mui/icons-material';
+import {
+  Autocomplete,
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  TextField,
+  Tooltip
+} from '@mui/material';
+
+import { resolve } from '../../api/api.js';
+import DatePickerField from '../date-picker-field/DatePickerField';
+import ConfirmationDialog from '../dialogs/ConfirmationDialog';
+import { AppContext } from '../../context/AppContext';
+import {
+  selectCsrfToken,
+  selectCurrentUser,
+  selectProfileMap
+} from '../../context/selectors';
+
+const organizationBaseUrl = '/services/volunteer/organization';
+const relationshipBaseUrl = '/services/volunteer/relationship';
+
+const formatDate = date => {
+  if (!date) return '';
+  if (date instanceof Date) return format(date, 'yyyy-MM-dd');
+  const paddedMonth = (date.$M + 1).toString().padStart(2, '0');
+  const paddedYear = date.$D.toString().padStart(2, '0');
+  return `${date.$y}-${paddedMonth}-${paddedYear}`;
+};
```

Comment:
Worthy of a date/format utility helper? Looks like there are three instances in the codebase now.

Reply:
Great callout! I moved this function to the existing file `src/helpers/datetime.js` and changed three components to use it. I just pushed a commit for this ... after testing it of course.